code/modules/cargo/console.dm

```diff
@@ -119,6 +185,8 @@
 			)
 		if((P.hidden && !(obj_flags & EMAGGED)) || (P.contraband && !contraband) || (P.special && !P.special_enabled) || P.DropPodOnly)
 			continue
+		if((P.permits & permits) != permits)
```

Comment:
@Ghommie this is the issue part
as this is, it blocks none of the crates
if its == it makes it makes all crates go away
idk how to fix this at all tried to get some but were at a dead end

Reply:
```suggestion
		if((P.permits & permits) != P.permits)
```